### ISSUE TITLE
`PlatformNotSupportedException: LocalDB is not supported on this platform.` when run in dev container

### ISSUE THREAD

**Issue by bamurtaugh** [dotnet-architecture/eShopOnWeb#884]
Thanks for adding a dev container to this repo! But when I try to run the app in the dev container, I get the following:

![image](https://user-images.githubusercontent.com/25310137/229677472-3911a0e5-195b-40de-aac5-68f918689bdf.png)

I researched this error and seems like it would be expected since this is a Linux-based container, but since others are using this dev container successfully, I'm guessing I'm missing something? Thanks! cc @timheuer since it looks like you added the initial dev container.

**Comment by bamurtaugh** [comment]
This appears related to https://github.com/dotnet-architecture/eShopOnWeb/issues/238.

**Comment by bamurtaugh** [comment]
I found https://github.com/dotnet-architecture/eShopOnWeb#configuring-the-sample-to-use-sql-server addressed this! So I'll close this issue.

I plan to open a subsequent PR to update the dev container config to follow the latest [spec](https://containers.dev/), and I can also add some info to the readme about using the dev container (including what I discovered here and some other various tips and tricks).
